server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MailboxFactoryTest.java

```diff
@@ -33,14 +33,19 @@
 import org.apache.james.mailbox.model.MailboxId;
 import org.apache.james.mailbox.model.MailboxPath;
 import org.apache.james.mailbox.store.SimpleMailboxMetaData;
+import org.assertj.core.api.JUnitSoftAssertions;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
 
 import com.google.common.collect.ImmutableList;
 
 public class MailboxFactoryTest {
     public static final char DELIMITER = '.';
 
+    @Rule
+    public final JUnitSoftAssertions softly = new JUnitSoftAssertions();
```

Comment:
https://joel-costigliola.github.io/assertj/assertj-core-features-highlight.html#soft-assertions

Reply:
And? This is an assertj object ...

```
import org.assertj.core.api.JUnitSoftAssertions;
```

Also it is what is used for soft assertions EVERYWHERE in the code and also is the safest and less noisy way to do soft assertions in the exact same link you gave :trollface: